formats/sarifutils/test_sarifutils.go

```diff
@@ -13,14 +13,16 @@ func CreateRunWithDummyResults(results ...*sarif.Result) *sarif.Run {
 	return run
 }
 
-func CreateRunWithDummyResultAndRuleProperties(property, value string, result *sarif.Result) *sarif.Run {
+func CreateRunWithDummyResultAndRuleMultipleProperties(result *sarif.Result, properties, values []string) *sarif.Run {
```

Comment:
The function name indicates that we have a function for multipleProperties, and a one for single.
If it's just that one function - I think you can keep the name.

Reply:
ok fixed